### ISSUE TITLE
Display data validator reports under user profile

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#1027]
**Display data validator reports under user profile**

I had to go back into my history to find this. Shouldn't we think about listing these in user profiles—even if it means prompting them somehow if they're not logged in?

-----
fbitem-4ff5cac51def768f0832dbfab51ea8a05863469d
User provided contact info: @kcopas
System: Safari 11.0.3 / Mac OS X 10.13.3
Referer: https://www.gbif.org/tools/data-validator/1516700048516
Window size: width 1214 - height 749
[API log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2018-02-08T08:13:24.646Z',mode:absolute,to:'2018-02-08T08:19:24.646Z'))&_a=(columns:!(_source),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
[Site log](http://elk.gbif.org:5601/app/kibana?#/discover?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2018-02-08T08:13:24.646Z',mode:absolute,to:'2018-02-08T08:19:24.646Z'))&_a=(columns:!(_source),index:'prod-portal-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499')),sort:!('@timestamp',desc)))
System health at time of feedback: CRITICAL

**Comment by MortenHofft** [comment]
Yes that would be useful. But I believe it was an active choice not to do so at this point. @thomasstjerne you know the reasoning better than I. I see 3 options

**The proper solution**
Would be to link the report to a user and have an API for retrieving them. But I'm certain there is no linkage between a validation report and a user at this point.

**Adding them to the user**
We could also add them to the user object, but I can't help thinking that would generate issues and noise. It doesn't really belong there.

**Saving them in local storage**
Not pretty and will break if multiple users are using the same machine and you cannot log in on another device to see your reports. But at least it won't clutter the user account object.
